src/internal/connector/exchange/exchange_data_collection.go

```diff
@@ -342,8 +341,8 @@ func getModTime(mt modTimer) time.Time {
 // responses into data.Stream items contained within the Collection
 type GraphSerializeFunc func(
 	ctx context.Context,
-	client *msgraphsdk.GraphServiceClient,
-	objectWriter *kioser.JsonSerializationWriter,
+	// TODO: Keepers: Not sure how to have the interface use Serialize without updating all the files.
+	service *graph.Service,
```

Comment:
This should use the `graph.Servicer` interface, not the `Service` object.  Go idioms: accept interfaces, return structs.  That way you can drop all the type identification.

Reply:
Updated [dd42105](https://github.com/alcionai/corso/pull/2045/commits/dd42105b3e21b897ee2309ae4839869abe1ae4d8)